Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ruby:2.3-alpine
+
+ENV APP_HOME /usr/src/metasploit-framework/
+WORKDIR $APP_HOME
+
+COPY Gemfile* m* Rakefile $APP_HOME
+COPY lib $APP_HOME/lib
+
+RUN apk update && \
+	apk add \
+  	ruby-bigdecimal \
+  	ruby-bundler \
+  	ruby-io-console \
+    autoconf \
+  	bison \
+  	subversion \
+  	git \
+  	sqlite \
+  	nmap \
+  	libxslt \
+  	postgresql \
+    ncurses \
+  && apk add --virtual .ruby-builddeps \
+    build-base \
+    ruby-dev \
+    libffi-dev\
+    openssl-dev \
+    readline-dev \
+    sqlite-dev \
+    postgresql-dev \
+    libpcap-dev \
+    libxml2-dev \
+    libxslt-dev \
+    yaml-dev \
+    zlib-dev \
+    ncurses-dev \
+    bison \
+    autoconf \
+  && bundle install \
```

Comment:
also if we add an `ARG BUNDLER_ARGS` we can pass some arguments to bundler from the docker-compose file by replacing the line with `bundle install $BUNDLER_ARGS`. We can also think about adding the `--system` option to bundler.